linux_os/guide/system/software/system-tools/package_openscap-scanner_installed/rule.yml

```diff
@@ -15,9 +15,9 @@ rationale: |-
 severity: medium
 
 identifiers:
-    cce@rhel6: 82978-8
-    cce@rhel7: 82977-0
-    cce@rhel8: 82976-2
```

Comment:
Please change this also in `rhel8/profiles/ospp.profile`.

Reply:
OK